src/main/java/org/gitlab/api/http/GitlabHTTPRequestor.java

```diff
@@ -270,12 +294,55 @@ private void findNextUrl() throws MalformedURLException {
         };
     }
 
+    private void submitAttachments(HttpURLConnection connection) throws IOException {
+        String boundary = Long.toHexString(System.currentTimeMillis()); // Just generate some unique random value.
+        connection.setDoOutput(true);
+        connection.setRequestProperty("Content-Type", "multipart/form-data; boundary=" + boundary);
+        String charset = "UTF-8";
+        String CRLF = "\r\n"; // Line separator required by multipart/form-data.
+        OutputStream output = connection.getOutputStream();
+        PrintWriter writer = new PrintWriter(new OutputStreamWriter(output, charset), true);
+        try {
+            for(Map.Entry<String, Object> paramEntry : data.entrySet()){
```

Comment:
Sorry! Hate being a jerk, but could you format this class too? Mostly just formatting, i.e. no space between `for` and the first parenthesis, `}finally{` etc